pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -30,7 +30,7 @@ func ProcessSearchTracesRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 	}
 
 	nowTs := putils.GetCurrentTimeInMs()
-	searchText, startEpoch, endEpoch, _, _, _ := pipesearch.ParseSearchBody(readJSON, nowTs)
+	searchText, startEpoch, endEpoch, _, _, _, _ := pipesearch.ParseSearchBody(readJSON, nowTs)
```

Comment:
Don't ignore the `error` field. Instead, check if it's not nil, and if so you should log an error message and return early

Reply:
Got it